### ISSUE TITLE
"Update Available" Rocket Chat (Rocket Cat) disabled, but still notifying

### ISSUE THREAD

**Issue by EqualMarcus** [RocketChat/Rocket.Chat#33126]
### Description:
Our setting for Rocket Chat "update available" notifications is off, but every time there's a new release, we have banners and chat notifications from Rocket Cat.

Also, when on 6.6, all of our users - most of whom are not tech savvy - were harassed with a big popup warning message. This is inappropriate, and should be only visible to admins.

### Steps to reproduce:

???

### Expected behavior:

Off means off for Rocket Cat

### Actual behavior:

Banners, Messages, etc.

### Server Setup Information:

- Version of Rocket.Chat Server: 6.9.2
- License Type: Enterprise
- Number of Users: 200+
- Operating System: Linux
- Deployment Method: Docker
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: 14
- MongoDB Version: 5?

### Client Setup Information

- Desktop App or Browser Version: Desktop 4.0.0 (Latest)
- Operating System: MacOS

### Additional context

N/A

### Relevant logs:

N/A

**Comment by reetp** [comment]
Yup, it has been complained about many times and I totally agree. 

Users absolutely do not need to see this. Admins only, at best, and the notifier Enable/Disable should work.

Couple of points to note.

You are on Enterprise so always contact support there first - it is what you pay for!! A complain there will have more wight than here.

You really do not want to be on your version.

I suggest you urgently upgrade to AT LEAST the latest 6.9.x (currently 6.9.5) and preferably the latest which is 6.11.1

https://github.com/RocketChat/Rocket.Chat/releases/tag/6.9.5

I am not sure what the status of the Enable/Disable is on 6.11.x - it needs testing.

**Comment by casalsgh** [comment]
Reetp shared with me and taking to internal teams for verification. This is a re-occurring situation and we'll work to get it resolved

**Comment by EqualMarcus** [comment]
Thanks for replying promptly 🙏 

1. Thanks - I'll be sure to contact support. Just my dev instincts to go to github haha
2. Yes thank you, we've updated to 6.9.2 - Just noticed the popup as we were on 6.6, and have now updated. The popup does work 🤣

**Comment by reetp** [comment]
> Yes thank you, we've updated to 6.9.2

>> You really do not want to be on your version.

>> I suggest you urgently upgrade to AT LEAST the latest 6.9.x (currently 6.9.5) and preferably the latest which is 6.11.1

>  (Latest using the Docker published by Docker, not Rocket Chat)

Why? The image there is not well maintained IIRC. 

There is at least one big security patch you should use. I'd do it yesterday if I were you.

Note. Do NOT use the "latest" tag either. Specify a version and upgrade on your own schedule. 

See here for why.

https://gist.github.com/reetp/ddd0580ab9f9397715da1c14cd85a73c#docker

Either way, you really should upgrade....

**Comment by LeeThompson** [comment]
This is still happening.   I'm running the latest 6.x (I can't run 7.x on the hardware it's on; I'm working on planning a hardware migration but mongodb is touchy so I'm taking my time and doing dry runs).   Every administrator (well, all 2 of us) gets nagged for every single update.  

Update notification is turned off.

This bug has been around for a long time now.

**Comment by LeeThompson** [comment]
Still happening in 7.5.1 _and_ now with the 7.6.2 nag, it can't be dismissed!

Update notification is still turned off; which it ignores.

**Please fix this!**

I stay a bit back in versions for a reason as you guys break thinks a lot and I like it wait for the point releases to stabilize.

**Comment by LeeThompson** [comment]
> > Please fix this!
> 
> You do not need to shout. You will just get ignored or blocked.
> 
> Rocket.Chat is open source, not free support and fixes on demand.
> 
> The team is aware of this as per Casals comment above, but they have varying priorities for paying customers who keep Rocket in business to enable them to write code.
> 
> If you have a paid version then open a support ticket with them and nag away.
> 
> If not you can either wait for a possible fix, or supply a PR with your fix which I am sure will be welcomed
> 
> Thanks.

First; I did not shout, I exclaimed.

Second; I don't need to be lectured, I'm aware of that but this issue has been open for nearly a year and while "tasked" there hasn't been any progress.    I've been coding for over 40 years and participated in open source projects, I know how it works.

Finally; All I wanted to do is try to ensure that this doesn't fall by the wayside.

**Comment by reetp** [comment]
> First; I did not shout, I exclaimed.

Ahhh that was it. Knew I missed something. I thought it was a demand.

> I know how it works.

In which case you don't need to say anything. Exclaimed, italicised, bold, or otherwise.

> All I wanted to do is try to ensure that this doesn't fall by the wayside.

As per my previous.

**Comment by LeeThompson** [comment]
> As per my previous.

There, I fixed it, now we can all get on with our lives.
